AutomatticTracks/src/test/java/com/automattic/android/tracks/crashlogging/SentryCrashLoggingTest.kt

```diff
@@ -51,13 +61,14 @@ class SentryCrashLoggingTest {
             shouldDropException = shouldDropException,
             extraKeys = extraKeys,
             provideExtrasForEvent = provideExtrasForEvent,
-            applicationContext = applicationContext,
+            initialApplicationContext = applicationContext,
         )
 
         crashLogging = SentryCrashLogging(
             context = mockedContext,
             dataProvider = dataProvider,
             sentryWrapper = mockedWrapper,
+            applicationScope = testScope.backgroundScope
```

Comment:
From the docs: 
<img width="675" alt="image" src="https://user-images.githubusercontent.com/5845095/181455644-c1ab0146-c35d-421f-bba5-8a67691ebc11.png">

It seems like a perfect use case for us as `SentryCrashLogging` starts 2 coroutines in its constructor (which don't stop).